test/screenshot/lib/screenshot.js

```diff
@@ -92,36 +100,69 @@ async function handleCaptureResponse(testPageUrl, captureResponseBody) {
   const isStillRunning = () => infoProgress.running > 0;
   const isTimedOut = () => (Date.now() - startTime) > API_MAX_WAIT_MS;
 
-  while (isStillRunning() && !isTimedOut()) {
+  do {
```

Comment:
Make this a function instead

Reply:
Instead of extracting a new function, I changed `isStillRunning()` to return `true` if `infoProgress` is `undefined`.